homeassistant/components/recorder/migration.py

```diff
@@ -1264,7 +1264,7 @@ def _migrate_columns_to_timestamp(
                     text(
                         "UPDATE events SET "
                         "time_fired_ts= "
-                        "(case when time_fired is NULL then 0 else EXTRACT(EPOCH FROM time_fired) end) "
```

Comment:
Needed to cast here in the case where schema changes gets deleted and they somehow managed to get this to run so it doesn't throw an sql error